modules/post-install-edge-aws-extend-cluster.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/aws-compute-edge-tasks.adoc
+
+:_content-type: CONCEPT
+[id="post-install-edge-aws-extend-cluster_{context}"]
+= Extend existing clusters to use AWS Local Zones
```

Comment:
There are two modules with this name? They don't conflict with one another, just pointing it out

Extend existing clusters to use AWS Local Zone